TravisBot/ViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  TravisBot
+//
+//  Created by Douglas Garcia Barroso on 22/03/20.
+//  Copyright © 2020 Douglas Garcia Barroso. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+
+}
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/dbgarcia/danger-bot/blob/4ad3f65193926f1211a9cfb888b46cbbba684b48/TravisBot/ViewController.swift#L20"></span>Files should have a single trailing newline.<br />
<code>trailing_newline</code> <code>ViewController.swift:20</code></del></td>
    </tr>
  </tbody>
</table>